src/flow/flow-fixtures.js

```diff
@@ -0,0 +1,56 @@
+// @flow
```

Comment:
I plan to add to this as necessary.  There is a follow-up task ([here](https://app.zenhub.com/workspace/o/uber-web/web-platform-tasks/issues/680)) to tackle the broader testing story for Flow.  For example, we only test the 'on-side' for now.